src/CommonITILValidation.php

```diff
@@ -67,9 +67,9 @@ public static function getIcon()
         return 'ti ti-thumb-up';
     }
 
-    public function getItilObjectItemType()
+    public static function getItilObjectItemType()
     {
-        return str_replace('Validation', '', $this->getType());
+        return str_replace('Validation', '', static::class);
```

Comment:
```suggestion
        return str_replace('Validation', '', static::getType());
```

Reply:
Is there some benefit of using `::getType`? Nothing overrides it and it just calls `get_called_class` making `::getType` just a slower version of `static::class`. I'm fairly certain this method only exists because GLPI predates static binding (PHP 5.3). The original implementation used the `type` property as a cache (not used anymore) and `get_class($this)`.

Reply:
The only case where it is maybe still required is for `Event` itemtype that was renamed to `Glpi\Event`. In the current case, it is probably not an issue to use `static::class`.

I will try to investigate to see if we can safely remove usages of `getType()`.